include/TrustWalletCore/TWCoinType.h

```diff
@@ -126,6 +126,7 @@ enum TWCoinType {
     TWCoinTypeAptos = 637,
     TWCoinTypeHedera = 3030,
     TWCoinTypeSecret = 529,
+    TWCoinTypeTheOpenNetwork = 607,
```

Comment:
CoinType is automatically generated as `TWCoinTypeTheOpenNetwork` from the blockchain name.
Is it correct? Should we rename it and related things to `TwCoinTypeTON`?

Reply:
Yes, you can rename to `TwCoinTypeTON`, just don't forget to change all the places

Reply:
> Yes, you can rename to `TwCoinTypeTON`, just don't forget to change all the places

Could you suggest what exactly should be renamed?
I have renamed every `TWCoinTypeTheOpenNetwork` in the code, however `CoinInfoData.cpp` is still generated with the old constant. We want to keep the blockchain called "The Open Network", but the coin name should be "TON".

Reply:
> > Yes, you can rename to `TwCoinTypeTON`, just don't forget to change all the places
> 
> Could you suggest what exactly should be renamed? I have renamed every `TWCoinTypeTheOpenNetwork` in the code, however `CoinInfoData.cpp` is still generated with the old constant. We want to keep the blockchain called "The Open Network", but the coin name should be "TON".

Looks like we misinterpreted "name" field, will rename blockchain and name properly.